opds_import.py

```diff
@@ -696,7 +696,9 @@ def run_once(self, start, cutoff):
 class OPDSImporterWithS3Mirror(OPDSImporter):
     """OPDS Importer that mirrors content to S3."""
 
-    def __init__(self, _db, **kwargs):
+    def __init__(self, _db, default_data_source, **kwargs):
```

Comment:
TestOPDSImporterWithS3Mirror doesn't actually instantiate this class (it instantiates OPDSImporter with a dummy S3Uploader object), so I didn't see that this constructor had the wrong signature.